Components/Image/image.html

```diff
@@ -49,8 +49,8 @@ <h1 class="section-header">Image</h1>
             <h3 class="section-sub-header">Responsive Image</h3>
             <p>This is used to display any image to the 100% of the parent container keeping the
                  aspect ratio in place. Use the classname responsive-image to achieve this in your project.</p>
-            <div class="flex-row">
-                <img src="../../Assets/Images/nature.jpg" alt="Nature"> 
+            <div class="flex-row image-width ">
+                <img class="responsive-image" src="../../Assets/Images/nature.jpg" alt="Nature"> 
```

Comment:
You  can add loading="lazy" in image tag.

Reply:
sure.